converter.py

```diff
@@ -0,0 +1,275 @@
+"""
+Convert existing jupyter notebook to an airbnb knowledge repo format
+"""
+import os
+import re
+import json
+import subprocess
+from dateutil import parser as date_parser
+
+
+def main(ml_repo, knowledge_repo, inplace):
+    ml_repo_path = os.path.abspath(ml_repo)
+    knowledge_repo_path = os.path.abspath(knowledge_repo)
+    if not os.path.isdir(knowledge_repo_path):
+        init_knowledge_repo(knowledge_repo_path)
+
+    convert_all_posts(ml_repo_path, knowledge_repo_path, inplace)
+
+
+def init_knowledge_repo(path):
+    cmd = 'knowledge_repo --repo {} init'.format(path)
+    subprocess.call(cmd, shell = True)
```

Comment:
`subprocess.call(cmd, shell=True)` (PEP8)

Reply:
Fixed in latest master, sorry for missing this ...